configure.ac

```diff
@@ -69,6 +76,7 @@ if test "x$MAKENSIS" = "x"; then
 fi
 AM_CONDITIONAL([HAVE_NSIS], [test "x$MAKENSIS" != "x"])
 
+if test "${enable_password_save}" = "yes"; then
```

Comment:
PKG_PROG_PKG_CONFIG may be also moved inside this "if then"  so that with --disable-password-change users can compile even if pkg-config is not available.